### PR TITLE
[0.5.1] l10n: sync de_DE es_ES fr_FR nb_NO nl pt_BR zh_Hant tr it_IT ar translations

### PR DESCRIPTION
**Pull request by dachary**
## Status

Ready for review

## Description of Changes

Update the latest translations from https://weblate.securedrop.club/

## Testing

These are new translations files (covered by a generic apparmor inclusion) and updates that can be verified if you speak the language by switching to the corresponding language using the menu. Ensuring there is no breakage because of placeholders incorrectly copied over can be done with:

* ./manage.py translate-messages --compile
* setting config.py line **SUPPORTED_LOCALES = ['ar','de_DE','es_ES','fr_FR','nb_NO','nl','pt_BR','zh_Hant','en_US','it_IT','tr']**
* running
<pre>
$ PAGE_LAYOUT_LOCALES='ar,de_DE,es_ES,fr_FR,nb_NO,nl,pt_BR,zh_Hant,it_IT,tr' pytest -v --page-layout tests/pages-layout
...
</pre>

## Deployment

The new files will be added when the package is updated. There is no action required from the sysadmin point of view.

### If you made changes to the app code:

- [x] Unit and functional tests pass on the development VM

### CODE REVIEW

**Comment by redshiftzero** [comment]
(Edited test plan to add `'ar'`)

**Comment by dachary** [comment]
@redshiftzero ah, thanks for the fix :-)

File: securedrop/translations/ar/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -0,0 +1,1350 @@
...
+"و&nbsp;تنصيب متصفّح تور:"
+
+#: source_templates/use-tor-browser.html:9
+msgid ""
+"If there is a chance that downloading the Tor Browser raises suspicion and "
+"your mail provider is less likely to be monitored, you can send a mail to "
+"<pre>gettor@torproject.org</pre> and a bot will answer with instructions."
+msgstr ""
+"إذا كان زيارة موقع تور لتنزيل المتصفح مثير للريبة، وحساب بريدك الإلكتروني "
+"غير مراقب يمكنك مراسلة <pre> 1gettor@torproject.org</pre> 2 وسوف تتلقى "
```
Comment (redshiftzero) [inline securedrop/translations/ar/LC_MESSAGES/messages.po]: Ah, this looks like a mistake: `<pre> 1gettor@torproject.org</pre> 2` -> `<pre> gettor@torproject.org</pre>`. Should we fix this in Weblate and then sync again to add that change to this PR @dachary?

**Comment by dachary** [comment]
Yes, I'll resync in 1 hour or so: https://forum.securedrop.club/t/finishing-touch-for-arabic-translation/350/5

**Comment by dachary** [comment]
Good catch !

**Comment by dachary** [comment]
sync with weblate & repushed

**Comment by dachary** [comment]
I'm very confused by this error https://circleci.com/gh/freedomofpress/securedrop/6640?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link, trying again

**Pull request merged**